### PR TITLE
Add wireframe

### PR DESCRIPTION
**Pull request by Arch-Noize**
Currently this is all just a small setup for the start of my project tomorrow!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,5 +25,8 @@
     "stylelint-scss": "^3.21.0",
     "webpack": "^5.88.2",
     "webpack-cli": "^5.1.4"
```
Comment (muneebulrehman) [inline package.json:27]: Kindly install `webpack-dev-server` dependency so as to start webpack server that will serve as the localhost.

File: src/index.html [hunk 2]
```diff
@@ -3,12 +3,26 @@
...
+            <h2>Add your score</h2>
+            <form class="container">
+                <input type="text" id="name" placeholder="Add your name..." autocomplete="off">
+                <input type="number" id="score" placeholder="Add your score..." autocomplete="off">
+                <button class="add">Add score</button>
+            </form>
+        </div>
     </div>
 </body>
 </html>
```
Comment (muneebulrehman) [inline src/index.html]: The UI should look like the provided wireframe design. However, in your case, the scores container is missing. Kindly add the same. 
- Optional. Consider populating some scores manually.

| Your design  | Wireframe design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Arch-Noize , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job setting up the project.

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly give a proper title along with the description of your project in the README file. Currently, the README has the title of your previous project along with the description. Remember, it is super important to have a professional README file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @Arch-Noize 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Configured `webpack` properly ✔️
- App layout is identical to the wireframe ✔️  

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**